remappings.txt

```diff
@@ -1,6 +1,6 @@
 @uniswap/v4-core/=lib/v4-core/
-forge-gas-snapshot/=lib/v4-core/lib/forge-gas-snapshot/src/
```

Comment:
installing directly here since it was removed from core. Will update to native snaps in separate PR